### PR TITLE
next release

### PR DESCRIPTION
**Pull request by PedroTroller**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,26 +17,27 @@
...
-        "symfony/http-foundation": "^5.4 || ^6.0",
-        "symfony/http-kernel": "^5.4 || ^6.0",
-        "symfony/validator": "^5.4 || ^6.0",
+        "php": ">=8.0",
+        "symfony/config": "^5.4 || ^6.2@dev",
+        "symfony/dependency-injection": "^5.4 || ^6.2@dev",
+        "symfony/form": "^5.4 || ^6.2@dev",
+        "symfony/http-foundation": "^5.4 || ^6.2@dev",
+        "symfony/http-kernel": "^5.4 || ^6.2@dev",
+        "symfony/validator": "^5.4 || ^6.2@dev",
```
Comment (AntoineLelaisant) [inline composer.json:26]: Not so sure that introducing a `x@dev` dependency is a good idea. Why the `^6.2` wouldn't be enough ?

File: src/Knp/DictionaryBundle/Dictionary/Invokable.php [hunk 2]
```diff
@@ -30,24 +28,12 @@ final class Invokable implements Dictionary
...
      */
-    public function __construct(string $name, callable $callable, array $callableArgs = [])
-    {
-        $this->name         = $name;
-        $this->callable     = $callable;
-        $this->callableArgs = $callableArgs;
-    }
-
-    public function getName(): string
+    public function __construct(private string $name, callable $callable, private array $callableArgs = [])
```
Comment (AntoineLelaisant) [inline src/Knp/DictionaryBundle/Dictionary/Invokable.php:34]: My PHP skills are a bit rusty ATM but couldn't you use `private callable $callable` here ?

File: composer.json (hunk 1 above)
Comment (PedroTroller) [reply composer.json:26]: To be compatible with next beta Symfony version.

File: src/Knp/DictionaryBundle/Dictionary/Invokable.php (hunk 2 above)
Comment (PedroTroller) [reply src/Knp/DictionaryBundle/Dictionary/Invokable.php:34]: Nope, it's not possible yet (I don't know why) to type a property as `callable`. But a method argument, yes.

**Review by AntoineLelaisant — APPROVED** [review]

**Pull request merged**